components/Searchbar/Searchbar.js

```diff
@@ -7,16 +7,19 @@ export default function SearchBar({ setSearchTerm }) {
       type="text"
       id="searchTerm"
       name="searchTerm"
-      placeholder="🔍 search... "
+      placeholder="Search... "
       aria-label="searchbar"
     />
   );
 }
 
 const SearchInput = styled.input`
   padding: 0.3rem;
-  font-size: 0.9rem;
+  font-size: 1rem;
```

Comment:
Oh wow. Look at this. You already use the relative unit for setting the font size .✨